kyaml/openapi/example_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package openapi_test
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/kustomize/kyaml/openapi"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+func Example() {
+	s := openapi.SchemaForResourceType(yaml.TypeMeta{APIVersion: "apps/v1", Kind: "Deployment"})
+
+	f := s.SchemaForField("spec").
```

Comment:
Nit, but I suspect this function would be called a lot, does it need to be that long?

Reply:
Done